es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,57 @@
+// const { createElement, noop } = require('../lib');
+// const TodoList = require('./todo-list');
+// const TodoForm = require('./todo-form');
+import noop from '../lib/noop.js';
+import createElement from '../lib/create-element';
```

Comment:
For imports from the `lib` folder they should be imported from the `lib/index.js` file, this allows for more flexibility and allows you to do multiple imports on a single line.
```JAVASCRIPT
import { noop, createElement } from '../lib';
```
Also if you just reference a folder it will default to the `index.js` file within that folder.